pennylane/compiler/python_compiler/transforms/mbqc/convert_to_mbqc_formalism.py

```diff
@@ -53,65 +53,43 @@
     "CNOT",
 }
 
+_MBQC_GATE_SET = _MBQC_ONE_QUBIT_GATES | _MBQC_TWO_QUBIT_GATES
```

Comment:
Minor but can we call this "_MBQC_GATES" instead of _GATE_SET? Just because we've already used "MBQC gate set" to mean the set of gates compatible with our MBQC execution, and that includes the Paulis.